tools/plugins/com.liferay.ide.server.tomcat.core/plugin.xml

```diff
@@ -331,5 +331,9 @@
             class="com.liferay.ide.server.tomcat.core.PortalTomcatBundleFactory"
             type="tomcat">
       </portalBundle>
+      <portalBundle
```

Comment:
Add this to the server core, no need to add it to tomcat.core.  Eventually tomcat.core will get moved into server.core in the future.